<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mp3Tag Qt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QToolBar" name="mainToolBar">
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="floatable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftPanel">
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QTreeView" name="fileTreeView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="fileInfoFrame">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Sunken</enum>
          </property>
          <layout class="QHBoxLayout" name="fileInfoLayout">
           <item>
            <widget class="QLabel" name="fileCountLabel">
             <property name="text">
              <string>Files: 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalSizeLabel">
             <property name="text">
              <string>Size: 0 KB</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="rightTabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tagTab">
        <attribute name="title">
         <string>Tag</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0" colspan="2">
          <widget class="QFrame" name="coverFrame">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>250</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>250</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Sunken</enum>
           </property>
           <widget class="QLabel" name="coverLabel">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>250</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="text">
             <string>No Cover</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="additionalGroupBox">
           <property name="title">
            <string>Additional Information</string>
           </property>
           <layout class="QFormLayout" name="additionalLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="trackLabel">
              <property name="text">
               <string>Track:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="trackEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="discLabel">
              <property name="text">
               <string>Disc:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="discEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="composerLabel">
              <property name="text">
               <string>Composer:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="composerEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="albumArtistLabel">
              <property name="text">
               <string>Album Artist:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="albumArtistEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="tagGroupBox">
           <property name="title">
            <string>Standard Tags</string>
           </property>
           <layout class="QFormLayout" name="standardTagsLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="titleLabel">
              <property name="text">
               <string>Title:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="titleEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="artistLabel">
              <property name="text">
               <string>Artist:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="artistEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="albumLabel">
              <property name="text">
               <string>Album:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="albumEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="yearLabel">
              <property name="text">
               <string>Year:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="yearEdit"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="genreLabel">
              <property name="text">
               <string>Genre:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="genreEdit"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="commentLabel">
              <property name="text">
               <string>Comment:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="commentEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="changeCoverButton">
           <property name="text">
            <string>Change Cover</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="fileTab">
        <attribute name="title">
         <string>File</string>
        </attribute>
        <layout class="QVBoxLayout" name="fileLayout">
         <item>
          <widget class="QGroupBox" name="fileInfoGroupBox">
           <property name="title">
            <string>File Information</string>
           </property>
           <layout class="QFormLayout" name="fileInfoFormLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="fileNameLabel">
              <property name="text">
               <string>File Name:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="fileNameValue">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="filePathLabel">
              <property name="text">
               <string>Path:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="filePathValue">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="fileSizeLabel">
              <property name="text">
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="fileSizeValue">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="fileTypeLabel">
              <property name="text">
               <string>Type:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="fileTypeValue">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="fileDurationLabel">
              <property name="text">
               <string>Duration:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="fileDurationValue">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="fileBitrateLabel">
              <property name="text">
               <string>Bitrate:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="fileBitrateValue">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="fileSampleRateLabel">
              <property name="text">
               <string>Sample Rate:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="fileSampleRateValue">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="fileChannelsLabel">
              <property name="text">
               <string>Channels:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="fileChannelsValue">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="playerTab">
        <attribute name="title">
         <string>Player</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QFrame" name="playerCoverFrame">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>250</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>250</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Sunken</enum>
           </property>
           <widget class="QLabel" name="playerCoverLabel">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>250</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="text">
             <string>No Cover</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="playerControlsGroupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="title">
            <string>Player Controls</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="2">
             <widget class="QPushButton" name="stopButton">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="currentFileValue">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QSlider" name="volumeSlider">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="currentFileLabel">
              <property name="text">
               <string>Current File:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="playbackPositionLabel">
              <property name="text">
               <string>Position:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="playbackDurationValue">
              <property name="text">
               <string>00:00</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="4">
             <widget class="QSlider" name="playbackSlider">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="playbackDurationLabel">
              <property name="text">
               <string>Duration:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="playButton">
              <property name="text">
               <string>Play</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="playbackStatusLabel">
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="playbackPositionValue">
              <property name="text">
               <string>00:00</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="playbackStatusValue">
              <property name="text">
               <string>Stopped</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pauseButton">
              <property name="text">
               <string>Pause</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
